webui/react/src/ioTypes.ts

```diff
@@ -268,6 +271,7 @@ export const ioExperimentConfig = io.type({
   resources: ioExpResources,
   searcher: io.type({
     metric: io.string,
+    name: io.keyof(experimentSearcherName),
```

Comment:
test suggestion: this should be running through our experiment config fixtures set. could you maybe add some configs that have names that are missing from our set?
the set currently has these
```
        "name": "adaptive",
        "name": "adaptive_asha",
        "name": "adaptive_simple",
        "name": "single",
```

Reply:
I looked as the file `src/fixtures/responses/experiment-details/set-a.json` and it has all 4 name types, so we should be good right?

Reply:
nm it's missing `grid`, `pbt`, and `random`

Reply:
I'm working with @katport to figure out fixtures for these 3, but I'll add them in as a separate task.

Reply:
I've added 3 exp config for `pbt`, `random` and `grid`